tests/utils.go

```diff
@@ -2092,17 +2092,18 @@ func NewRandomFedoraVMIWithDmidecode() *v1.VirtualMachineInstance {
 }
 
 func GetGuestAgentUserData() string {
+	guestAgentUrl := GetUrl(GuestAgentHttpUrl)
 	return fmt.Sprintf(`#!/bin/bash
                 echo "fedora" |passwd fedora --stdin
                 mkdir -p /usr/local/bin
-                sleep 1
+                for i in {1..20}; do curl -I %s | grep "200 OK" && break || sleep 0.1; done
```

Comment:
ahh you did it... please squash this with the previous commit.

Reply:
Done.